examples/README.md

```diff
@@ -0,0 +1,12 @@
+# Examples
+
+This is a collection of examples that shows how to interact with Astarte in common use cases.
+
+## Javascript examples
+
+- [javascript-sensors-viewer](javascript-sensors-viewer): sample web page in vanilla Javascript to
```

Comment:
Javascript or JavaScript? Which is the correct one?

Reply:
Thanks for the heads up, we will stick to JavaScript. I fixed all occurrences of it in the PR.